### ISSUE TITLE
Can't build the app because make erros concerning the lib

### ISSUE THREAD

**Issue by felipejoribeiro** [oblador/react-native-store-review#69]
Error message:

```
version is 1.1.15.
e: /Users/fejori/Work/snackin/snackin/node_modules/react-native-store-review/android/src/main/java/com/oblador/storereview/StoreReviewModule.kt: (16, 16): Unresolved reference: mutableMapOf
e: /Users/fejori/Work/snackin/snackin/node_modules/react-native-store-review/android/src/main/java/com/oblador/storereview/StoreReviewModule.kt: (16, 56): Unresolved reference: to
e: /Users/fejori/Work/snackin/snackin/node_modules/react-native-store-review/android/src/main/java/com/oblador/storereview/StoreReviewPackage.kt: (16, 16): Unresolved reference: emptyList

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-store-review:compileDebugKotlin'.
> Compilation error. See log for more details

```

package.json:

```
{
  ...
  "resolutions": {
    "styled-components": "^5"
  },
  "dependencies": {
    "@amplitude/react-native": "^2.16.1",
    "@react-native-async-storage/async-storage": "^1.17.10",
    "@react-native-clipboard/clipboard": "^1.11.1",
    "@react-native-community/netinfo": "^9.3.6",
    "@react-native-picker/picker": "^2.4.8",
    "@react-navigation/bottom-tabs": "^6.4.0",
    "@react-navigation/native": "^6.0.13",
    "@react-navigation/native-stack": "^6.9.1",
    "@react-navigation/stack": "^6.3.3",
    "@sentry/react-native": "^4.7.1",
    "async-mutex": "^0.4.0",
    "base-64": "^1.0.0",
    "credit-card-type": "^9.1.0",
    "date-fns": "^2.29.3",
    "email-validator": "^2.0.4",
    "events": "^3.3.0",
    "jwt-decode": "^3.1.2",
    "lottie-react-native": "^5.1.4",
    "node-cache": "^5.1.2",
    "react": "17.0.2",
    "react-native": "0.68.1",
    "react-native-animatable": "^1.3.3",
    "react-native-animated-pagination-dot": "^0.4.0",
    "react-native-background-timer": "^2.4.1",
    "react-native-bootsplash": "^4.3.2",
    "react-native-camera": "^3.40.0",
    "react-native-check-version": "^1.0.20",
    "react-native-config": "^1.4.11",
    "react-native-confirmation-code-field": "^7.3.1",
    "react-native-device-info": "^10.3.0",
    "react-native-geolocation-service": "^5.3.1",
    "react-native-gesture-handler": "^2.8.0",
    "react-native-image-picker": "^4.10.0",
    "react-native-maps": "^1.3.2",
    "react-native-masked-text": "^1.13.0",
    "react-native-modal": "^13.0.1",
    "react-native-onesignal": "^4.4.1",
    "react-native-permissions": "^3.6.1",
    "react-native-picker-select": "^8.0.4",
    "react-native-safe-area-context": "^4.4.1",
    "react-native-screens": "^3.18.2",
    "react-native-snackbar": "^2.4.0",
    "react-native-sound": "^0.11.2",
    "react-native-store-review": "^0.2.1",
    "react-native-svg": "^13.4.0",
    "react-native-svg-transformer": "^1.0.0",
    "react-native-version-number": "^0.3.6",
    "react-native-view-pdf": "^0.14.0",
    "react-native-webview": "^11.23.1",
    "react-native-wifi-reborn": "^4.7.0",
    "socket.io-client": "^4.5.3",
    "styled-components": "^5.3.6"
  },
  "devDependencies": {
    "@babel/core": "^7.12.9",
    "@babel/runtime": "^7.12.5",
    "@react-native-community/eslint-config": "^2.0.0",
    "babel-jest": "^26.6.3",
    "eslint": "^7.32.0",
    "jest": "^26.6.3",
    "metro-react-native-babel-preset": "^0.67.0",
    "react-test-renderer": "17.0.2"
  },
  "jest": {
    "preset": "react-native"
  }
}

```

**Comment by oblador** [comment]
Hi! Please try using React Native 0.70 or a previous version of this lib.

**Comment by felipejoribeiro** [comment]
Thank you for the prompt response.
I tried these versions:

"react-native-store-review": "0.3.0",
"react-native-store-review": "0.2.1",
"react-native-store-review": "0.2.0",
"react-native-store-review": "0.1.5",

The last one did it. Thanks.

It's strange because we were using version 0.2.1 previously, and we updated our react-native version from 0.63.3 to 0.68.0 when the build error started occurring. Is there any possibility that something in our build.gradlew could be causing this version freeze?
